bigquery/storage/managedwriter/appendresult.go

```diff
@@ -174,7 +180,9 @@ func newPendingWrite(appends [][]byte) *pendingWrite {
 				},
 			},
 		},
-		result: newAppendResult(appends),
+		result:       newAppendResult(appends),
+		attemptCount: 1,
```

Comment:
This `attemptCount` attribute is not being used anywhere yet. Is this for the future `AppendRetryer` interface ?

Reply:
Yes, the top description notes it's being added but not used yet.